### PR TITLE
Extend and document the profile object

### PR DESCRIPTION
**Pull request by cjbarth**
This relates to the discussion in #278 .

### CODE REVIEW

**Comment by cjbarth** [comment]
This may also address #256

File: README.md [hunk 1]
```diff
@@ -73,6 +73,26 @@ passport.use(new MultiSamlStrategy(
...
+  spNameQualifier?: string;
+  mail?: string;
+  email?: string;
+  getAssertionXml(): string;
+  getAssertion(): object;
+  ID?: string;
+}
+```
+
 #### Config parameter details:
```
[the remark this answers is not in the document]
Comment (cjbarth) [reply README.md]: @markstos I believe I've addressed your concerns here. Please let me know if there is more to do.

**Comment by markstos** [comment]
@cjbarth Everything looks great now. Merging.

**Comment by ray2k** [comment]
When might the npm package get updated to include this (specifically, getAssertion)? I installed latest but get 0.35.0 which according to readme is current latest and greatest, and only the getAssertionXml is there.

**Comment by markstos** [comment]
@ray2k This week, I think. We are packaging some breaking changes in a bundle, so trying go about it carefully. Install from Github if you want, expecting it will be forward compatible. NPM's package.json has a syntax for that.

**Pull request merged**